IpmiFeaturePkg/Library/IpmiCommandLib/IpmiCommandLibNetFnSensor.c

```diff
@@ -0,0 +1,83 @@
+/** @file
+  IPMI Command - NetFnSensor.
+
+  Copyright (c) Microsoft Corporation
+  PDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#include <Library/BaseMemoryLib.h>
+#include <Library/DebugLib.h>
+#include <Library/IpmiBaseLib.h>
+
+#include <IpmiFeature.h>
+
+/**

```

Comment:
I suggest updating the function descriptions to match our newer style (i.e. drop `@brief`, add `[in]`/`[out]` to params, etc.)